### ISSUE TITLE
SKS intercepts Google searches

### ISSUE THREAD

**Issue by m-schmitt** [arnemart/SafariKeywordSearch#41]
I went to Google.com, entered _A Discovery of Witches_ in the Google search box, but instead of getting the Google search results, it took me to Amazon. I finally figured out that it was because the search phrase started with "A" and "A" is the keyword for an Amazon search.

So the question is, why is SKS getting involved?

My theory is...

1. Entering something in the Google search box and hitting enter results in a URL such as https://www.google.com/search?q=search+terms
2. Which is the same as what Safari would do if you entered the search terms in the Safari URL field
3. Which is the signal to SKS that you entered something, that it should parse for a keyword

In other words, SKS can't tell the difference between keywords entered in the URL field or in the Google search box. If Google is the default search engine, but it would the same for any other default search engine.

**Comment by LeanderBlume** [comment]
This same issue makes setting a default search engine in SKS basically useless. *Every* query entered on google.com is directly piped into the engine set as default.
